### PR TITLE
Add an example to base_label API

### PR DESCRIPTION
**Pull request by fxn**
Easy one.

### CODE REVIEW

File: vm_backtrace.c [hunk 1]
```diff
@@ -315,9 +315,10 @@ location_base_label(rb_backtrace_location_t *loc)
 }
 
 /*
- * Returns the base label of this frame.
+ * Returns the label of this frame without decoration.
  *
- * Usually same as #label, without decoration.
+ * For example, if the label is `foo`, this method returns `foo`, same, but if
```
Comment (byroot) [inline vm_backtrace.c:320]:
```suggestion
 * For example, if the label is `foo`, this method returns `foo` as well, but if
```

Comment (byroot) [reply vm_backtrace.c:320]: Maybe it would be worth to take the same example from `#label`?

**Comment by fxn** [comment]
@byroot I am looking at the implementation of `location_label` and it does a lot of things, I am not confident "Returns the label of this frame without decoration" is accurate. But "usually" as it was originally written is not good documentation either, you have to tell me what does the method return.

**Comment by fxn** [comment]
@byroot In order to be more accurate with confidence, I should dig into this. By now, I have restored the "usually", so the patch does not change the documented relationship between `label` and `base_label`.

**Comment by byroot** [comment]
> I should dig into this

No problem. Your patch look fine to me, but just let me know when your are done.

**Comment by fxn** [comment]
@byroot yes, let's keep this patch as it is, and if in the future I dig into the implementation of `label` and may submit a followup.

**Pull request merged**